src/platyPS/platyPS.psm1

```diff
@@ -748,6 +748,8 @@ function New-ExternalHelp
         [string[]]$ApplicableTag,
 
         [System.Text.Encoding]$Encoding = [System.Text.Encoding]::UTF8,
+
+        [string]$OutputFile,
```

Comment:
I don't think `OutputFile` is the most suitable name. This is more like log file. Maybe something like `ErrorLogFile` ?

Reply:
I will change.